checkout-core/src/main/java/com/adyen/checkout/core/util/LocaleUtil.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2020 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by caiof on 17/12/2020.
+ */
+package com.adyen.checkout.core.util
+
+import android.content.Context
+import android.os.Build
+import java.util.Locale
+import java.util.MissingResourceException
+
+/**
+ * Utility class to use [Locale].
+ */
+object LocaleUtil {
+
+    /**
+     * Get the current user Locale.
+     * @param context The context
+     * @return The user Locale
+     */
+    @JvmStatic
+    fun getLocale(context: Context): Locale {
+        return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            context.resources.configuration.locales[0]
+        } else {
+            @Suppress("DEPRECATION")
+            context.resources.configuration.locale
+        }
+    }
+
+    /**
+     * Gets the language tag from a Locale.
+     *
+     * @param locale The locale.
+     * @return The tag of the locale, or null if not valid.
+     */
+    @JvmStatic
+    fun toLanguageTag(locale: Locale): String {
+        return locale.toLanguageTag()
```

Comment:
What is this method doing now? Seems redundant.

Reply:
It is redundant but I thought it's better to have our wrapper methods around the `Locale` methods, in case we want to change the way we serialize and deserialize locales. What do you think?

Reply:
OK makes sense